src/shims/unix/solarish/foreign_items.rs

```diff
@@ -28,7 +29,22 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriInterpCxExt<'mir, 'tcx> {
             // Incomplete shims that we "stub out" just to get pre-main initialization code to work.
             // These shims are enabled only when the caller is in the standard library.
             "stack_getbounds" if this.frame_in_std() => {
-                let [_stack] = this.check_shim(abi, Abi::C { unwind: false }, link_name, args)?;
+                let [stack] = this.check_shim(abi, Abi::C { unwind: false }, link_name, args)?;
+                let stack = this.deref_pointer_as(stack, this.libc_ty_layout("stack_t"))?;
+                // Not sure we can figure out ss_flags field ?
```

Comment:
Is there documentation somewhere on what that field means?

Reply:
[yes](https://docs.oracle.com/cd/E36784_01/html/E36874/stack-getbounds-3c.html)

Reply:
Ah, [found it](https://docs.oracle.com/cd/E36784_01/html/E36874/stack-getbounds-3c.html)

> The ss_flags member is set to SS_ONSTACK if the calling thread is executing on its alternate signal stack, and zero otherwise.

So, you can just set it to 0, since we don't support signals. (But add a comment explaining why we set it to 0.)

Then you can also remove the `write_bytes_ptr` as we are then setting all fields.

And you can remove the `if this.frame_in_std()` as this is actually doing its job then. :D  (okay the stack address is nonsense but so far that hasn't caused issues)